epochStart/metachain/auctionListDisplayer.go

```diff
@@ -0,0 +1,202 @@
+package metachain
+
+import (
+	"encoding/hex"
+	"fmt"
+	"math/big"
+	"strconv"
+	"strings"
+
+	"github.com/ElrondNetwork/elrond-go-core/core"
+	"github.com/ElrondNetwork/elrond-go-core/display"
+	logger "github.com/ElrondNetwork/elrond-go-logger"
+	"github.com/ElrondNetwork/elrond-go/state"
+)
+
+const maxPubKeyDisplayableLen = 20
+const maxNumOfDecimalsToDisplay = 5
+
+func (als *auctionListSelector) displayMinRequiredTopUp(topUp *big.Int, startTopUp *big.Int) {
```

Comment:
could be extracted also to a new struct: auctionListDisplayer that does the displaying.

Reply:
I think it's better to leave it as it is now, since I will probably split the code anyway in a future PR which requires to create API data for auction config.

Reply:
ok